red4ext-sys/src/interop.rs

```diff
@@ -311,8 +404,25 @@ unsafe impl ExternType for VoidPtr {
     type Kind = cxx::kind::Trivial;
 }
 
+pub struct ResourcePathBuilder {
```

Comment:
I think we should move it to the main crate, there's no good reason for either this or the macro to be in the sys crate. The sys crate is dedicated for native bindings

Reply:
If you change the `new` method to accept a Path-like argument like the other comment mentions I think the builder might become a little pointless because Path provides equivalent API already, so we might no longer need this